working-groups/README.md

```diff
@@ -36,6 +36,12 @@ AMP’s Working Groups are:
   * Facilitator: @dvoytenko
   * [Approvers WG Github repo](https://github.com/ampproject/wg-approvers)
   
+* **Bento.**
+  * Response for...
```

Comment:
@dvoytenko what's this group responsible for?  (ideally without using the term "Bento" :))

Reply:
The group is responsible for evaluating, designing, and implementing the architecture in which AMP elements can be used without AMP Runtime, and/or within popular Web frameworks.